sys/net/gnrc/network_layer/icmpv6/error/gnrc_icmpv6_error.c

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (C) 2015 Martine Lenders <mlenders@inf.fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
+ * @file
+ */
+
+#include "kernel_macros.h"
+#include "net/gnrc/pktbuf.h"
+
+#include "net/gnrc/ipv6/netif.h"
+#include "net/gnrc/icmpv6/error.h"
+#include "net/gnrc/icmpv6.h"
+
+/* all error messages are basically the same size and format */
+#define ICMPV6_ERROR_SZ (sizeof(icmpv6_error_dst_unr_t))
+#define ICMPV6_ERROR_SET_VALUE(data, value) \
+    ((icmpv6_error_pkt_too_big_t *)(data))->mtu = byteorder_htonl(value);
+
+static inline size_t _fit(gnrc_pktsnip_t *pkt)
+{
+    /* TODO: replace GNRC_IPV6_NETIF_DEFAULT_MTU with known path MTU? */
+    return MIN((gnrc_pkt_len(pkt) + ICMPV6_ERROR_SZ), GNRC_IPV6_NETIF_DEFAULT_MTU);
```

Comment:
Can `GNRC_IPV6_NETIF_DEFAULT_MTU` be greater than `IPV6_MIN_MTU`? RFC requires it must be less than `IPV6_MIN_MTU` https://tools.ietf.org/html/rfc4443#section-2.4